sdk-python/omnilogger/get_pricing.py

```diff
@@ -38,8 +39,8 @@ def load_llm_pricing():
     return pricing
 
 
-def read_json_at(path: str) -> dict:
-    with open(path, "r") as json_file:
+def read_json_at(path: str) -> dict[str, Any]:
```

Comment:
aren't all dicts `dict[str, x]`? 🤔 why was this change necessary

Reply:
```
dict --> dict[Unknown, Unknown]

you can have 
dict[str,Any]
dict[Any,str]

```